controllers/socket/broadcaster.go

```diff
@@ -38,6 +39,16 @@ func StopBroadcaster() {
 
 func SendMessage(steamid string, event string, content string) {
 	broadcastMessageChannel <- broadcastMessage{
+		Room:    "",
+		SteamId: steamid,
+		Event:   event,
+		Content: content,
+	}
+}
+
+func SendMessageToRoom(room string, steamid string, event string, content string) {
```

Comment:
You're sending to a room so you don't need a steam id. Use `socketServer.BroadcastTo`. (btw, socket.BroadcastTo broadcasts the message to everyone except the socket: socket.Emit + socket.BroadcastTo = server.BroadcastTo)